### PR TITLE
Fix for streaming table modifications.

### PR DESCRIPTION
**Pull request by a-hansen**
The user has confirmed these work.

### CODE REVIEW

**Comment by a-hansen** [comment]
@Pacane Please review.

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java [hunk 1]
```diff
@@ -173,10 +167,67 @@ public synchronized void setStreaming(int rid,
...
+            }
+            if (streamMutex == null) {
+                streamMutex = new Object();
+            }
+            mutex = streamMutex;
+        }
+        synchronized (mutex) {
+            if (writer == null) {
+                try {
+                    if (millis <= 0)
```
Comment (Pacane) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: Missing brackets.

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java [hunk 2]
```diff
@@ -173,10 +167,67 @@ public synchronized void setStreaming(int rid,
...
+                        mutex.wait();
+                    else
```
Comment (Pacane) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: same

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java [hunk 3]
```diff
@@ -226,13 +277,35 @@ public synchronized JsonObject getTableMeta() {
...
+    /**
      *
      * @return Columns of the table.
      */
     public synchronized List<Parameter> getColumns() {
         return columns != null ? Collections.unmodifiableList(columns) : null;
     }
 
+    private void setColumns(List<Parameter> cols) {
+        if (cols == null) {
```
Comment (Pacane) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: cols -> columns

**Comment by Pacane** [comment]
LGTM after comments

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java (hunk 1 above)
Comment (a-hansen) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: Fixed

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java (hunk 2 above)
Comment (a-hansen) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: done

File: sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java (hunk 3 above)
Comment (a-hansen) [inline sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java]: Fixed

**Comment by Pacane** [comment]
@a-hansen Looks like the CI passed :)

**Pull request merged**